File: src/main/java/org/example/models/Item.java
package org.example.models;

public class Item {
    private String name;
    private boolean isReserved;

    public Item(String name) {
        this.name = name;
        this.isReserved = false;
    }

    public String getName() {
        return name;
    }

    public boolean isReserved() {
        return isReserved;
    }

    public void setReserved(boolean reserved) {
        this.isReserved = reserved;
    }

    @Override
    public String toString() {
        return name + (isReserved ? " (Réservé)" : " (Disponible)");
    }
}

File: models/Item.php
<?php

class Item {
    private $name;
    private $isReserved;

    public function __construct($name) {
        $this->name = $name;
        $this->isReserved = false;
    }

    public function setReserved($reserved) {
        $this->isReserved = $reserved;
    }

    public function getName() {
        return $this->name;
    }

    public function isItemReserved() {
        return $this->isReserved;
    }
}
?>
